src/pages/docs/guides/chat/build-task-oriented.mdx

```diff
@@ -0,0 +1,385 @@
+goes down, it has little-to-no impact on the global service and your application.
+
+## Priced for task-oriented efficiency
+
+Task-oriented chats are typically short-lived but can be numerous, making efficient pricing crucial for cost management.
+
+**Consumption-based pricing** is ideal for most task-oriented scenarios:
+
+* **Pay per message/minute:** Only pay for actual engagement, perfect for sporadic task conversations
+* **Volume discounts:** Lower rates as your usage scales across all your tasks
+* **No user-based fees:** You're not charged for users who join briefly or tasks that end quickly
+
+**MAU-based pricing** works well when you have:
+
+* **Consistent task volume:** Regular support tickets, frequent deliveries, or ongoing collaborative work
+* **Predictable user engagement:** Teams or customers who regularly participate in tasks
+* **Longer-term task relationships:** Ongoing gaming communities or service subscriptions
+
+For example:
+- **Support platforms:** Often benefit from consumption pricing due to variable ticket volume
+- **Delivery services:** MAU pricing if you have consistent delivery personnel and customers
+- **Gaming platforms:** Consumption pricing for casual games, MAU for subscription-based gaming
+
+A full breakdown of pricing options, including a cost estimator tailored to your task-oriented scenario, can be found on the [pricing page](/pricing).
+
+### Aggressive connection management
+
+When a client abruptly disconnects from Ably, there is a 2 minute delay before the connection is cleaned up on the server, to enable the client to resume the connection from where it left off. When you're finished with an Ably connection, be sure to call the `close()` method to gracefully shut down the connection.
+
+All Ably SDKs also perform "heartbeats" with the server to enable detection of dropped or disrupted connections. The default interval for this is 15 seconds. By adjusting the heartbeat interval, you can control how quickly a connection is deemed to have dropped and therefore reduce the amount of time connections remain open for.
+
+<Code>
+```javascript
+import * as Ably from 'ably';
+
+const realtimeClient = new Ably.Realtime({
+   // Other options omitted for brevity
+   transportParams: { heartbeatInterval: 5000 }
+})
+```
+</Code>
+
+## Push notifications
+
+Ably Pub/Sub channels can be used for push notifications. See the [Push notifications](/docs/push/publish) documentation for more details. Since Ably Chat rooms are backed by a single Ably Pub/Sub channel, you can use the same channel for push notifications to notify all participants in the room when something happens.
+
+You can also use a separate channel to control notifications, in which case you can tailor them to individual users. Read more about push notifications with Ably in the [Push notifications](/docs/push/publish) documentation.
+
+## Enriching tasks with Ably's realtime services
```

Comment:
I think we should have this much higher up the document, as it's a big USP compared to other "chat providers" - Ably doesn't just do Chat, it does everything else realtime associated with these sorts of activities.